### PR TITLE
Soarian FHIR R4 Observation Resouorce

### PR DESCRIPTION
**Pull request by DeepakGupta93**
Description
----
Review Request for Soarian R4 Observation Resouorce

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
Observation_1
Observation_2
Observation_3
Observation_4
Observation_Headers
Observation_5
Observation_Errors

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/clinical/diagnostics/observation.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Parameters
+
+ Name          | Required?           | Type            | Description
+---------------|---------------------|-----------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This, or `patient`  | [`token`]       | The logical resource id associated with the resource.
```
Comment (HRNWriter) [inline content/soarian/r4/clinical/diagnostics/observation.md:74]: In the first row of the Description, change "id" to "ID".

PRODOC-160857

File: layouts/soarian/r4/clinical/diagnostics.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+
+<p>The Diagnostic resources are concerned with the process of observing, testing, and diagnosing a patient. Note, once a diagnosis is confirmed, it would be recorded as a Condition so that it is part of the historical patient record.</p>
```
Comment (HRNWriter) [inline layouts/soarian/r4/clinical/diagnostics.html:3]: In the second sentence, please change "Note," to "Note:" (Change the comma to a colon.)

PRODOC-160857

File: layouts/soarian/r4/clinical/diagnostics.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+<p>The Observation resource is primarily concerned with recording discrete values, answers, and results such as clinical laboratory data and vital signs while the DiagnosticReport resource typically provides a textual set of information and interpretation after performing a diagnostic service such as a Radiology report or Pathology report.</p>
```
Comment (HRNWriter) [inline layouts/soarian/r4/clinical/diagnostics.html:5]: Break this statement into two sentences:
The Observation resource is primarily concerned with recording discrete values, answers, and results such as clinical laboratory data and vital signs.
The DiagnosticReport resource typically provides a textual set of information and interpretation after performing a diagnostic service such as a Radiology report or Pathology report.</p>
 
PRODOC-160857

**Review by HRNWriter — COMMENTED** [review]
Please see my comments.

**Comment by sonikrish** [comment]
Can you please attach screenshots for where headers (accept type) and error is rendered. Apart from that everything looks good to me. +1 once the snapshot is attached.

File: content/soarian/r4/clinical/diagnostics/observation.md (hunk 1 above)
Comment (DeepakGupta93) [reply content/soarian/r4/clinical/diagnostics/observation.md]: Changed id to "ID"

File: layouts/soarian/r4/clinical/diagnostics.html (hunk 2 above)
Comment (DeepakGupta93) [reply layouts/soarian/r4/clinical/diagnostics.html]: Changed "," to colon ":"

File: layouts/soarian/r4/clinical/diagnostics.html (hunk 3 above)
Comment (DeepakGupta93) [reply layouts/soarian/r4/clinical/diagnostics.html]: Made changes as suggested

File: layouts/soarian/r4/clinical/diagnostics.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+<p>Click <a href="https://hl7.org/fhir/r4/diagnostics-module.html">here</a> to see the detailed categorization and full set of resources that fall into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.</p>
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply layouts/soarian/r4/clinical/diagnostics.html]: Changed this as "Select"

**Review by slvsridhar — APPROVED** [review]

**Review by vf056083 — APPROVED** [review]

**Review by HRNWriter — COMMENTED** [review]
Robin's "Observation.code" section still needs to be added after the "Terminology Bindings" section.
Was the entire "Observation.component.code" section removed? Only the lines for "Details: CPT" were supposed to be removed.
PRODOC-171149

**Review by VamsiMure — APPROVED** [review]

**Comment by PBKarthik** [comment]
After Publish
image
image
image

**Pull request merged**